cafebabel/templates/articles/detail.html

```diff
@@ -24,14 +24,16 @@ <h1>{{ article.title }}</h1>
                     <li class=translated-language><a href={{ translation.detail_url }}>{{ translation.language }}</a></li>
                 {% endfor %}
             {% endif %}
-            {% for language in get_languages() %}
-                {% if article.language != language[0] and language[0] not in translations_langs %}
+            {% for lang_code, lang_label in get_languages() %}
+                {% if article.language != lang_code and lang_code not in translations_langs %}
                     {% if article.is_translation %}
-                        {% if language[0] != article.original_article.language %}
-                            <li class=to-translate-languages><a href={{ url_for('translations.create') }}?lang={{language[0]}}&original={{ article.original_article.id }}>{{ language[1] }}</a></li>
+                        {% if lang_code != article.original_article.language %}
+                            <li class=to-translate-languages><a href={{ url_for('translations.create') }}?lang={{lang_code}}&original={{ article.original_article.id }}>{{ lang_label }}</a></li>
```

Comment:
I know it was already the case but let's try to be consistent and add spaces around vars in templates: `{{ lang_code }}` (same below)